server/src/main/java/org/elasticsearch/action/search/SearchPhaseExecutionException.java

```diff
@@ -134,11 +134,10 @@ private static String buildMessage(String phaseName, String msg, ShardSearchFail
     @Override
     protected void metadataToXContent(XContentBuilder builder, Params params) throws IOException {
         builder.field("phase", phaseName);
-        final boolean group = params.paramAsBoolean("group_shard_failures", true); // we group by default
-        builder.field("grouped", group); // notify that it's grouped
+        builder.field("grouped", true); // notify that it's grouped
```

Comment:
I assume this is only here for backwards compatibility?

Reply:
yea I did not want to change the output format at this stage. Also, given that the flag was not exposed, we would always return true anyways even before the change. We should remove this at some point though.

Reply:
👍